frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1911,7 +1911,7 @@ With this feature enabled, the current page is factored in, resulting in the cou
             },
             {
                 text_func = function()
-                    return T(_("Container bottom margin: %1"), self.settings.container_bottom_padding)
+                    return T(_("Items mini-bar bottom margin: %1"), self.settings.container_bottom_padding)
```

Comment:
There doesn't seem to be much reason to specify anything here.
```suggestion
                    return T(_("Bottom margin: %1"), self.settings.container_bottom_padding)
```

<img src=https://github.com/koreader/koreader/assets/202757/c7526ff3-1fe6-4520-a45f-06a8a8e6e682 width=40%> <img src=https://github.com/koreader/koreader/assets/202757/94bc0f0a-0a41-435f-9356-67af8a85c724 width=40%>

Reply:
mmm, I didn't think of that, might be a bit of a stretch but I guess you are right.

Reply:
In terms of what you call the invisible box I think it's strictly speaking inaccurate. I believe the height setting can add a top and bottom margin and this setting adds yet *another* bottom margin underneath all that, but I think being slightly inaccurate may well be a lot clearer here than being totally accurate to some obscure implementation details.